pages/linux/pkgfile.md

```diff
@@ -2,7 +2,7 @@
 
 > Tool for searching files from packages in the official repositories on arch-based systems.
 > See also `pacman files`.
```

Comment:
Is this how we refer to other tldr commands? It took me some time to understand that the author meant to run `tldr pacman files`

Reply:
We use it in many pages (84 to be exact), so it's pretty standardized. I personally think it's fairly clear, but writing ```See also: `tldr pacman files` ``` wouldn't hurt to avoid confusion. Let's see what the others think.

Reply:
I also did not check beforehand that you @marchersimon was the author.
I was happy enough that was able to open two separate pull requests simultaneously :) one step at a time...

Reply:
I'm not _totally_ sure that prepending `tldr` won't _add_ to any confusion, but I'm not sure.  I don't have much of an opinion, although I am slightly biased toward it staying out. 🤷🏻‍♂️

Reply:
What about 
> See also `pacman files` page.

Denoting a word *page* would make it more clear *to me* (no sure about anyone else) that I am not supposed to run `pacman files` (there is obviously no such option, because there is `pacman --files`), nor look into `man X pacman`, where X is a specific number.

This detail is applicable only to the tldr pages for the subcommands, and only to such pages that have it as a parameter requiring a single or double dash not just a word.

An example where this does not apply is `pio boards`, because is also available as both, a platformio subcommand and a tldr subcommand page. By running it I get a hint that I can see details about it in tldr as well.

It could maybe also be formulated more like
> See also `pacman files`, a page describing `pacman --files`.

These are all just my thoughts, please mark this as resolved anytime you see fit.

Reply:
Maybe it is even worth add a reference into issue #5299 (I did not want to clog it right there) as I believe an alias mechanism would solve two issues ensuing from this comment thread:

1. more obvious, subcommand aliases from `tldr pio` to `tldr platformio` as not all subcommands have the alias reference (as `tldr platformio` has), for instance `tldr platformio boards` is missing (maybe all of them are missing)

2. more subtle, the page `pacman files` should be aliased with `pacman --files` and the *See also* section should refer to `pacman --files` as it is the actual command that is  being run

Reply:
Hrm. I suggest here that we include the name of the thing that you do `tldr {{thing}}` to get the page here, as the usual flow is

1. See "See Also `thing`"
2. Type `tldr thing`

If this isn't clear, we generally add an extra clause to the sentence. For example, something similar to your suggestion above would work well:

```markdown
> See also `pacman files`, describing the usage of `pacman --files`.
```